programs/context_meta.zasm

```diff
@@ -3,6 +3,12 @@
 	.globl	__entry
 __entry:
 .func_begin0:
+	add 42, r0, r1
+	; store 42 in heap incrementing by 33 bytes
+	st.1 r0, r1
+	; store in aux heap incrementing by 33 bytes
+	st.2 r0, r1
```

Comment:
For that increment you'd be missing the `.inc` modifier I believe. Also, isn't it 32 bytes?

Reply:
>For that increment you'd be missing the .inc modifier I believe.

It goes without any modifier. If your doubt comes from the fact that the spec document lists [`OpStore`](https://matter-labs.github.io/eravm-spec/spec.html#:~:text=in2%20aliased%20as-,st.1.inc%20in1%2C%20out,-uma.aux_heap_write%20in1) using the `inc` modifier, I believe it's an error, as it would make it identical to [`OpStoreInc`](https://matter-labs.github.io/eravm-spec/spec.html#:~:text=in2%20aliased%20as-,st.1.inc%20in1%2C%20out,-uma.inc.aux_heap_write).

>Also, isn't it 32 bytes?

Correct! it appears as there was a small error on our codebase which I interpreted wrongly, it's now fixed.